recipe-server/normandy/settings.py

```diff
@@ -405,6 +419,10 @@ class ProductionReadOnly(Production):
         'security.W017',  # Check CSRF cookie http only
     ])
 
+    # In ReadOnly mode you have no business using the API to do any writes
+    # anyway. This gives early feedback to that.
+    CORS_ALLOW_METHODS = values.ListValue(['GET', 'OPTIONS'])
```

Comment:
Since this is controllable via envvars, it should also be in `Base`, so it is available to all environments. It should also be in the documentation.